### ISSUE TITLE
"Suggest" triggers in non-focused group, causing focus to shift away from current group

### ISSUE THREAD

**Issue by garyking** [Microsoft/vscode#57234]
- VSCode Version: Version 1.26.1 (1.26.1)
- OS Version: 10.13.5 (17F77)

Steps to Reproduce:

This is a continuation of #47858, another issue of mine from April. I was unable to figure out how to reproduce it, until now.

The following steps are also illustrated in a GIF below.

1. Have two tab groups of the same file. 
2. Create a JS file with these contents:

```
function join(value1, value2) {
  return value1 + value2
}

join(value1, value2)
```

3. In the first group, delete the `value2` until it's `va`, so that the tooltip appears. 
4. Switch to the second group. The tooltip will then close, as expected.
5. In the second group, modify the function params, such as by deleting some text.
6. The tooltip suddenly appears in group 1, taking focus away from group 2. If you were to be typing something at the time, then once focus is switched to group 1, the subsequent text you type will be entered there instead.

Maybe don't allow non-focused groups to show Suggest/IntelliSense tooltips? Or don't allow them to take away focus.

![tab group focus](https://user-images.githubusercontent.com/343940/44631637-e2065d80-a93c-11e8-866e-a5ad9be3045c.gif)

Does this issue occur when all extensions are disabled?: Yes

**Comment by ramya-rao-a** [comment]
This looks similar to #54545

Can you try our [Insiders](https://code.visualstudio.com/insiders) and see if you see the same there?

**Comment by garyking** [comment]
Yes seems fixed in Insiders, after copying the steps I mentioned above. I notice the change was made on August 6, yet 1.26 was released on August 14. Why did the change not make it into that release? I assume it will be in the next one?

**Comment by ramya-rao-a** [comment]
Code freeze for 1.26 was July 30 and so this fix didn't make it to the 1.26 release
Yes, it will be available in the next update

**Comment by ramya-rao-a** [comment]
Thanks @garyking
